partiql-cli/src/bin/partiql.sublime-syntax

```diff
@@ -0,0 +1,131 @@
+%YAML 1.2
```

Comment:
I'm curious if there a better place to store these syntax files so they could be used by other programs. Is there a way to publish the sublime syntaxes (i.e. partiql.sublime-syntax and ion.sublime-syntax) somewhere similar to what you had done for the ACE editor (https://github.com/ajaxorg/ace/pull/4630)? Alternatively, we could have a separate repo for the syntax highlighting modes that could be taken as a submodule of this repo.

Reply:
Yes. We should likely have all the tooling somewhere else and bring it in via submodule or something else here. The official way add a sublime extension is via https://packagecontrol.io/docs/submitting_a_package